### PR TITLE
P0 image

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

File: css/components/image.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+.img-square-round{
+    border-radius: 50%;
+}
+
+.img-landscape-round{
+    overflow: hidden;
+    border-radius: 50%;
+}
+
+.img-landscape-round>img{
```
Comment (manojsarna) [inline css/components/image.css:19]: looks good!!

File: css/components/image.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+width: 16rem;
+}
+
+.img-l{
+width: 32rem;
+}
+
+.img-xl{
+width: 64rem;
+}
```
Comment (manojsarna) [inline css/components/image.css:39]: ok great!

File: css/components/input.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.input{
+  border:none;
+  border-bottom: 3px solid ;
+  padding: 0.5rem;
+}
+
```
Comment (manojsarna) [inline css/components/input.css]: check px usage

File: css/utility/font.css [hunk 4]
```diff
@@ -11,8 +31,32 @@
...
+.fw-reg{
+  font-weight: 400;
+}
+
+.fw-med{
+  font-weight: 600;
+}
+
+.fw-b{
+  font-weight: 700;
```
Comment (manojsarna) [inline css/utility/font.css]: looks good!!

**Review by manojsarna — APPROVED** [review]
Overall looks good to me!!

you can merge it

File: css/utility/reset.css [hunk 5]
```diff
@@ -87,14 +88,6 @@ ul,ol{
...
 }
 
-.img-responsive{
-  width:100%;
-  height:auto;
-}
-
-.img-round{
-  border-radius: 50%;
-}
```
Comment (deekshithmd) [inline css/utility/reset.css:95]: code looks good to go

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**